### ISSUE TITLE
add() got an unexpected keyword argument 'input_shape'

### ISSUE THREAD

**Issue by pengweimin** [keras-team/keras#12968]
the code run in laptop is ok,
win10, 
tensorflow1.13 
keras2.2.4
python3.7
-----------------------------------------------------------------------
but in computer is error.
add() got an unexpected keyword argument 'input_shape'
win 10
tensorflow-gpu1.4
keras 2.2.4
python 3.5.2
---------------------------------------------------------------------------
the complete code: 
import numpy as np

data=np.empty((1,3,1000))
label=np.empty(1)
for i in range(1,10):
    arr=np.load("B"+str(i)+".npz")
    data=np.concatenate((data,arr['x_data']))
    label=np.concatenate((label,arr['y_data']))
data=data[1:,:,:]
data=data.transpose(0,2,1)#data.shape(None,1000,3)timespets=1000,channels=3
label=label[1:]

from keras.models import Sequential
from keras.layers import Dense, Activation, Flatten,LSTM,Permute,Reshape
from keras.layers import BatchNormalization, Dropout, Conv2D, MaxPooling2D
from sklearn.model_selection import KFold
from sklearn.model_selection import train_test_split
import matplotlib.pyplot as plt
from keras import backend as K

K.tensorflow_backend._get_available_gpus()
kf = KFold(n_splits=5)

for train, test in kf.split(data):
    x_train, x_test, y_train, y_test = train_test_split(

        data, label, test_size=0.1, random_state=42)

    model = Sequential()

    model.add(LSTM(128),input_shape=(x_train.shape[1],x_train.shape[2]))
    model.add(Activation('relu'))
    model.add(Dropout(0.2))
    model.add(Dense(2,activation='softmax'))
    print(model.summary())
    model.compile(loss = 'sparse_categorical_crossentropy',
                  optimizer = 'adam',
                  metrics = ['accuracy'])
    history=model.fit(x_train, y_train,validation_data=(x_test, y_test),
              epochs = 100, 
              batch_size = 256)
---------------------------------------------------------------------------

**Comment by jvishnuvardhan** [comment]
@pengweimin There is a typo in one line as follows

`model.add(LSTM(128,input_shape=(x_train.shape[1],x_train.shape[2])))`

Please check GitHub gist that I created for mnist data with your code ( as I don't have your data). Thanks!

**Comment by jvishnuvardhan** [comment]
Automatically closing this out since I understand it to be resolved, but please let me know if I'm mistaken.Thanks!
